src/databricks/labs/ucx/assessment/queries/README.md

```diff
@@ -0,0 +1,6 @@
+# Naming convention
+
+All files in this directory follow the virtual grid of a dashboard:
+
+* total width is 6 columns
+* all files are named as `<row>_<column>_something.sql`
```

Comment:
So, do we deploy new widgets by using the file name and not by using the -- viz and -- widget line ? we are still using _parse_magic_comment to do so

Reply:
for now, yes - later we can add more magic there :) I'll copy your comment to this readme

Reply:
Ah, ok, yes, that makes sense. Answers my question earlier. +1

Reply:
Conversely though I think we should stop using the filename in the query object ID, and maybe use a simple parser that removes the numbers, splits the rest on underscores and then title-cases the words. That way the names are more readable.